### PR TITLE
Guide users to create a first node pool after cluster creation

### PR DESCRIPTION
**Pull request by kuosandys**
Towards https://github.com/giantswarm/giantswarm/issues/18219.

This PR adds prompts in several different places to guide users to create a first node pool after cluster creation.

#### Cluster creation form:
Screen Shot 2021-11-17 at 13 54 48
This is similar to the message we currently show in cluster details > worker nodes tab (no node pools placeholder).

#### Flash message on successful cluster creation:
Screen Shot 2021-11-17 at 13 49 28

#### Cluster details > worker nodes widget:
Screen Shot 2021-11-17 at 13 50 04
Clicking the button brings users to the worker nodes tab with the node pool creation form open.

### CODE REVIEW

File: src/components/MAPI/workernodes/ClusterDetailWidgetWorkerNodes.tsx [hunk 1]
```diff
@@ -136,12 +137,27 @@ const ClusterDetailWidgetWorkerNodes: React.FC<IClusterDetailWidgetWorkerNodesPr
...
+            align='end'
+          >
+            <Box>
+              <Text margin={{ bottom: 'small' }}>No node pools</Text>
+              <Text size='small'>Create node pools to run workloads.</Text>
+            </Box>
+            <StyledLink
+              to={{
+                pathname: workerNodesPath,
+                state: { hasNoNodePools: true },
```
Comment (axbarsan) [inline src/components/MAPI/workernodes/ClusterDetailWidgetWorkerNodes.tsx:154]: Nice 👍

File: src/components/MAPI/workernodes/ClusterDetailWidgetWorkerNodes.tsx [hunk 2]
```diff
@@ -136,12 +137,27 @@ const ClusterDetailWidgetWorkerNodes: React.FC<IClusterDetailWidgetWorkerNodesPr
...
+              }}
+            >
+              <Button icon={<i className='fa fa-add-circle' />} tabIndex={-1}>
```
Comment (axbarsan) [inline src/components/MAPI/workernodes/ClusterDetailWidgetWorkerNodes.tsx:157]:
```suggestion
              <Button icon={<i className='fa fa-add-circle' role='presentation' aria-hidden='true' />} tabIndex={-1}>
```

**Review by axbarsan — APPROVED** [review]
LGTM

**Comment by marians** [comment]
Great!

**Pull request merged**